### PR TITLE
[FIX] sale_product_configurator: fix alignment value and radio button

### PR DESCRIPTION
**Pull request by ppr-odoo**
- Issue:
   Currently there is no alignment between radio button and value,
   we fix this commit.

- Fix:
   We move the input box inside the label.

- Step:
    - Install sale app
    - Create product(test) and add Attributes & Variants
    - Select Brand  (Display Type: radio) Attributes and add value
    - Create sale order and add product(test)
    - Check Configure

### CODE REVIEW

**Review by Demesmaeker — CHANGES_REQUESTED** [review]
Hello,

Small bug here, when I select "Aluminium", it doesn't work any more. Well, it's actually aluminium, but it doesn't show. Same with other attribute value. See screenshot:
image

edit: 
Before for comparison:
image

**Comment by ppr-odoo** [comment]
> Hello,
> 
> Small bug here, when I select "Aluminium", it doesn't work any more. Well, it's actually aluminium, but it doesn't show. Same with other attribute value. See screenshot: image
> 
> edit: Before for comparison: image

I fixed issue [link](https://tinyurl.com/2a2v8v44) and update the commit msg. please look at.thanks

**Review by Demesmaeker — APPROVED** [review]

**Review by chevalierv — APPROVED** [review]

File: addons/test_sale_product_configurators/static/tests/tours/product_configurator_edition_ui.js [hunk 1]
```diff
@@ -40,13 +40,13 @@ registry.category("web_tour.tours").add('sale_product_configurator_edition_tour'
 }, {
     trigger: '.fa-pencil',
 }, {
-    trigger: 'tr:has(.o_sale_product_configurator_name:contains("Customizable Desk"))>td:nth-child(2)>ul:nth-child(3)>li:nth-child(2)>div label:contains("Aluminium") ~ input:checked',
+    trigger: 'tr:has(.o_sale_product_configurator_name:contains("Customizable Desk"))>td:nth-child(2)>ul:nth-child(3)>li:nth-child(2)>div label:contains("Aluminium") > input',
```
Comment (Feyensv) [inline addons/test_sale_product_configurators/static/tests/tours/product_configurator_edition_ui.js:43]: Why did you remove the `:checked` part of the trigger ? :think

Comment (ppr-odoo) [reply addons/test_sale_product_configurators/static/tests/tours/product_configurator_edition_ui.js]: @Feyensv @Demesmaeker Done Sorry for that we forgot. Thanks

**Comment by Feyensv** [comment]
@robodoo r+
